json-spec/examples/landsat8-sample.json

```diff
@@ -35,92 +35,195 @@
  
   "properties": {
     "datetime": "2014-06-02T09:22:02Z",
-    "provider": "LANDSAT_8",
+    "provider": "USGS",
     "license": "PDDL-1.0",
-
-    "eo:off_nadir_angle" : -0.001,
+    "eo:platform": "landsat-8",
+    "eo:instrument": "OLI_TIRS",
+    "eo:collection": "L1T",
```

Comment:
I suggest using a different name for the collection id, 'L1T' is used as a processing level identification for products, see https://landsat.usgs.gov/landsat-processing-details

I miss the previous ls:data_type item that represented what seemed to me the item processing level. For CBERS for example that would be 'L2' for system correction or 'L4' for ortho. It seems to me that we could have Items with distinct processing levels in the same Collection, so this info should be in the Item level.

Reply:
@fredliporace The collection could be based on processing level. For instance landsat surface ref could be a separate collection. CBERS L2 and L4 might be different collections.

We have talked about processing level, but the problem is there is no standard for processing levels. Everyone uses different codes, numbers, and definitions.  That's why having a Collection represent a specific product (e.g., ortho'd product)

Reply:
@fredliporace - I think  you should still use 'data_type', just do it as 'cbers:data_type'. One of the main ideas behind stac is try to only specify as required the practices that are in place. The ls: was already supposed to just be a ls: thing. For cbers put in all the fields you think are useful for users to search / index, in whatever way you want.

@matthewhanson - let's be sure to not say definitively that 'collection represents a specific product', as providers organize things in different ways. The 'collection' should just be how the provider wants to define it. CBERS can have one collection with different levels specified as types, Planet can have one big 'planetscope' collection with different assets generated on the fly for different levels, and Landsat can have its collections be processing levels. Though someone else could even choose to set up a single big landsat 'collection' that uses a 'type' of some sort to configure.

Clearly collection is an important concept for imagery, so it's worth having, but we should just define it as the set of common metadata and assets that a provider chooses to use to divide things up.

Reply:
Sure, I agree with all that. A provider can define it however they want, I think you hit upon the major distinction between collections: "as the set of common metadata and assets that a provider chooses to use to divide things up."  However they decide to do that is fine.